api/lib/application/certification-center-memberships/certification-center-membership-controller.js

```diff
@@ -40,6 +51,17 @@ const update = async function (
   );
 };
 
-const certificationCenterMembershipController = { disable, update };
+const disable = async function (request, h, dependencies = { requestResponseUtils }) {
+  const certificationCenterMembershipId = request.params.id;
+  const pixAgentUserId = dependencies.requestResponseUtils.extractUserIdFromRequest(request);
+
+  await usecases.disableCertificationCenterMembership({
+    certificationCenterMembershipId,
+    updatedByUserId: pixAgentUserId,
+  });
+  return h.response().code(204);
+};
+
+const certificationCenterMembershipController = { updateFromPixCertif, updateFromPixAdmin, disable };
```

Comment:
```suggestion
const certificationCenterMembershipController = { disable, updateFromPixAdmin, updateFromPixCertif };
```
Si tu choisis de prendre en compte la suggestion, serait-il possible de faire la même réorganisation avec les fonctions de la ligne 6 à 63 ?